src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -166,10 +169,16 @@ export const WorkspaceDashboard = _.flow(
           authorizationDomain, createdDate, lastModified, bucketName,
           attributes, attributes: { description = '' }
         }
-      }
+      }, authState: { profile }, ajax: { User }
     } = this.props
-    const { submissionsCount, storageCostEstimate, editDescription, saving, consentStatus } = this.state
+    const { submissionsCount, storageCostEstimate, editDescription, saving, savingProfile, consentStatus, wsNotifications } = this.state
     const isEditing = _.isString(editDescription)
+    const notificationDetails = {
+      'Workspace changed': {
+        label: 'Data added or removed',
+        tooltip: 'Notify me when a Writer or Owner adds or changes data within this workspace'
```

Comment:
I think the tooltip might be better more helpful on the text rather than the box

Reply:
Agreed!